### DISCUSSION TITLE
Error when manually entering CDU wattage in 23.04

### DISCUSSION THREAD

**Discussion by ccowan-raeden** [opendcim/openDCIM#1549] [General]
After upgrading to 23.04, an error occurs when updating a new CDU Wattage via Manual Entry. Other CDUs that had a value set prior to upgrading still display properly but cannot be updated. 

The following error is logged when updating the device.

`PHP Fatal error:  Uncaught Error: Attempt to assign property "Wattage" on null in /var/www/openDCIM-23.04/devices.php:555\nStack trace:\n#0 {main}\n  thrown in /var/www/openDCIM-23.04/devices.php on line 555, referer:`

**Comment by MathisICDC** [comment]
I had the same problem with this version.
In spite of everything, the injection of this data by api still works.
A ticket has been created for this in the errors found: #1552
Mathis
